.github/workflows/dependency-submission.yml

```
@@ -0,0 +1,30 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install dependencies (ci install for full lock resolution)
+        working-directory: src
+        run: npm ci
+
+      - name: Submit npm Dependencies
+        uses: actions/dependency-review-action@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.12)

[error] input "token" is not defined in action "actions/dependency-review-action@v4". available inputs are "allow-dependencies-licenses", "allow-ghsas", "allow-licenses", "base-ref", "comment-summary-in-pr", "config-file", "deny-groups", "deny-licenses", "deny-packages", "external-repo-token", "fail-on-scopes", "fail-on-severity", "head-ref", "license-check", "repo-token", "retry-on-snapshot-warnings", "retry-on-snapshot-warnings-timeout", "show-openssf-scorecard", "show-patched-versions", "vulnerability-check", "warn-on-openssf-scorecard-level", "warn-only"

(action)